templates/vertical-full-page-map/markup/map.hbs

```diff
@@ -1 +1 @@
-<div id="js-answersMap" class="Answers-map js-answersMap"></div>
+<div id="js-answersMap" class="Answers-map js-answersMap" dir="ltr"></div>
```

Comment:
we don't want to flip the map for rtl?

Reply:
it mess with and pin placement (specifically the coordinate to css styling translation) on the map

Reply:
does this cause the map to not be oriented correctly, for either google/mapbox? Like disregarding pin stuff, does anything else change if you add dir="ltr" on a rtl page

Reply:
the map is oriented the same way for both ltr and rtl, only the pins are affected. the div that I added `dir='ltr'` only contains map related things like the canvas for the map and the pin clusters which we don't want rtl on. There's also the map control html elements but the position for those are specify for ltr and rtl in the map provider construction code. everything else on the page would follow ltr. tldr: nothing else change when I looked at the pages

Reply:
got it awesome!!